### DISCUSSION TITLE
Cannot read properties of null (reading 'blocks')

### DISCUSSION THREAD

**Discussion by fernandofas** [gatsbyjs/gatsby#37309] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I'm using https://github.com/gatsbyjs/gatsby-starter-wordpress-homepage and after the set up the following message come sup on the screen:  Cannot read properties of null (reading 'blocks')

Any idea how to sort this out as I'm running out of ideas and stuck on this issues that the tutorials are never clear about.

Kind regards,

Fernando

### Reproduction Link

https://cantdoit.com

### Steps to Reproduce

1. install https://github.com/gatsbyjs/gatsby-starter-wordpress-homepage
2. follow the README.MD steps
3. gatsby develop
...

### Expected Result

I would expect the page described in the example to show up, but that's not what is happening.

### Actual Result

Cannot read properties of null (reading 'blocks')

./src/pages/index.js:13

> 13 |       {homepage.blocks.map((block) => {
     |                ^

### Environment

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
    CPU: (8) x64 Intel(R) Core(TM) i7-4770K CPU @ 3.50GHz
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 18.12.1 - ~/.nvm/versions/node/v18.12.1/bin/node
    npm: 8.19.2 - ~/.nvm/versions/node/v18.12.1/bin/npm
  Browsers:
    Chrome: 108.0.5359.124
  npmPackages:
    gatsby: ^5.0.0 => 5.3.2
    gatsby-plugin-image: ^3.0.0 => 3.3.2
    gatsby-plugin-manifest: ^5.0.0 => 5.3.1
    gatsby-plugin-sharp: ^5.0.0 => 5.3.2
    gatsby-plugin-vanilla-extract: ^4.0.0 => 4.0.0
    gatsby-source-contentful: ^8.0.0 => 8.3.1
    gatsby-source-filesystem: ^5.0.0 => 5.3.1
    gatsby-source-wordpress: ^7.3.2 => 7.3.2
    gatsby-transformer-sharp: ^5.0.0 => 5.3.1
  npmGlobalPackages:
    gatsby-cli: 5.3.1
```

**Comment by tyhopp** [comment]
Hi, have you followed all the steps in https://github.com/gatsbyjs/gatsby-starter-wordpress-homepage/tree/c1111a1ea80e4f86b8666b7bed18a34aa3bd9b06#quick-start, particularly step 2, "Import content to your WordPress instance"?

The starter expects your Wordpress instance to have content blocks that it can render, so you'll need to create them.
